decks.js

```diff
@@ -319,6 +319,102 @@ echo $message;`,
                 seenHint: false
             }
         ]
+    },
+    {
+        name: "Fruits",
+        instructions: "The emoji/s on each card are a clue for a fruit. You can flip (click) the card to see an additional hint. Drag and drop a card onto the matching text clue on the right, which will be the fruit name in easy mode, and a fact of the food in the challenging mode. You can make a maximum of 3 wrong matches.",
+        cards: [
+            {
+                name: "Orange",
+                content: {
+                    primary: "O 🏃‍♂️ G", 
+                },
+                hint: "a colour",
+                matchValue: "Orange is a fruit and a color lol",
+                seenHint: false
+            },
+            {
+                name: "Pear",
+                content: {
+                    primary: "P 👂"
+                },
+                hint: "green color fruit",
+                matchValue: "it has about 100 calories",
+                seenHint: false
+            },
+            {
+                name: "Cherry",
+                content: {
+                    primary: "🪑 E"
+                }
```

Comment:
You missed the comma at the end on line 340, take care of such minor syntax errors in the future!